doc/en/how-to/usage.rst

```diff
@@ -35,11 +35,13 @@ Pytest supports several ways to run and select tests from the command-line.
 
 .. code-block:: bash
 
-    pytest -k "MyClass and not method"
+    pytest -k 'MyClass and not method'
 
 This will run tests which contain names that match the given *string expression* (case-insensitive),
 which can include Python operators that use filenames, class names and function names as variables.
 The example above will run ``TestMyClass.test_something``  but not ``TestMyClass.test_method_simple``.
+For the default windowns terminal ``cmd.exe`` you should use double quotes i.e. ``"``. For all other
+shells ``'`` is preferred.
```

Comment:
I think it would be nice to have this comment as a Sphinx "note". Also some wording suggestions:

```suggestion

.. note::

    If you are using the Windows command interpreter ``cmd.exe`` to run pytest, you should use double quotes, as in ``"MyClass and not method"``.
    In most other shells, ``'`` is preferred.
```